set.mm

```diff
@@ -662596,6 +663406,1223 @@ closed under the multiplication ( ' X ' ) of a finite number of
+       ` A ` , or ` F ` converges to ` A ` .  (Contributed by Glauco
+       Siliprandi, 2-Jan-2022.) $)
+    climuzlem $p |- ( ph -> ( F ~~> A <-> ( A e. CC /\ A. x e. RR+ E. j e. Z A.
+       k e. ( ZZ>= ` j ) ( abs ` ( ( F ` k ) - A ) ) < x ) ) ) $=
+      ( cc wcel cv wral crp wa cz cvv adantr cli wbr cfv cmin cabs clt cuz wrex
+      co climcl adantl id climrel brrelexi eqidd clim mpbid simprd simpr rexuz3
+      wi wb syl mpbird ralimi reximi a1i mpd ralimdva jca simprl nfv w3a uzssz2
+      ex nfre1 sseli 3ad2ant2 simpll uztrn2 adantll ffvelrnda syl2anc rspe 3exp
+      3impia rexlimd ralimdv imp adantrl fvexi fexd impbida ) AFCUAUBZCLMZENZFU
+      CZCUDUIUEUCBNZUFUBZEDNZUGUCZOZDHUHZBPOZQZAWNQZWOXDWNWOACFUJUKXFWQLMZWSQZE
+      XAOZDRUHZBPOZXDWNXKAWNWOXKWNWNWOXKQZWNULWNBCWQDEFSFCUAUMUNWNWPRMZQWQUOUPU
+      QURUKAXKXDVAWNAXJXCBPAXJXCVAWRPMAXJXCAXJQZXIDHUHZXCXNXOXJAXJUSAXOXJVBZXJA
+      GRMXPIXHDEGHJUTVCTVDXOXCVAXNXIXBDHXHWSEXAXGWSUSVEVFVGVHVOTVITVHVJAXEQZWNX
+      LXQWOXKAWOXDVKAXDXKWOAXDXKAXCXJBPAXBXJDHADVLXIDRVPAWTHMZXBXJAXRXBVMWTRMZX
+      IXJXRAXSXBHRWTGHJVNVQVRAXRXBXIAXRQZWSXHEXAXTWPXAMZQZWSXHYBWSQXGWSYBXGWSYB
+      AWPHMZXGAXRYAVSXRYAYCAGWPWTHJVTWAAHLWPFKWBWCTYBWSUSVJVOVIWFXIDRWDWCWEWGWH
+      WIWJVJAWNXLVBXEABCWQDEFSAHLSFKHSMAHGUGJWKVGWLAXMQWQUOUPTVDWM $.
+  $}
+
+  ${
+    $d A i j k l x y $.  $d F i j l x y $.  $d M i $.  $d Z i j l x y $.
+    $d i l ph y $.
+    climuz.k $e |- F/_ k F $.
+    climuz.m $e |- ( ph -> M e. ZZ ) $.
+    climuz.z $e |- Z = ( ZZ>= ` M ) $.
+    climuz.f $e |- ( ph -> F : Z --> CC ) $.
+    $( Express the predicate:  The limit of complex number sequence ` F ` is
+       ` A ` , or ` F ` converges to ` A ` .  (Contributed by Glauco
+       Siliprandi, 2-Jan-2022.) $)
+    climuz $p |- ( ph -> ( F ~~> A <-> ( A e. CC /\ A. x e. RR+ E. j e. Z A. k
+       e. ( ZZ>= ` j ) ( abs ` ( ( F ` k ) - A ) ) < x ) ) ) $=
+      ( vl vi cv cfv cmin clt wral nfcv vy cli wbr cc wcel co cabs cuz wrex crp
+      wa climuzlem wb wceq breq2 ralbidv rexbidv fveq2 raleqdv nffv nfov oveq1d
+      nfbr nfv fveq2d breq1d cbvral a1i bitrd cbvrexv cbvralv anbi2i ) AFCUBUCC
+      UDUEZMOZFPZCQUFZUGPZUAOZRUCZMNOZUHPZSZNHUIZUAUJSZUKZVMEOZFPZCQUFZUGPZBOZR
+      UCZEDOZUHPZSZDHUIZBUJSZUKZAUACNMFGHJKLULWEWQUMAWDWPVMWCWOUABUJVRWJUNZWCVQ
+      WJRUCZMWASZNHUIZWOWRWBWTNHWRVSWSMWAVRWJVQRUOUPUQXAWOUMWRWTWNNDHVTWLUNZWTW
+      SMWMSZWNXBWSMWAWMVTWLUHURUSXCWNUMXBWSWKMEWMEVQWJREVPUGEUGTEVOCQEVNFIEVNTU
+      TEQTECTVAUTERTEWJTVCWKMVDVNWFUNZVQWIWJRXDVPWHUGXDVOWGCQVNWFFURVBVEVFVGVHV
+      IVJVHVIVKVLVHVI $.
+  $}
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+                           Inferior limit (lim inf)
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c liminf $.
+
+  $( Extend class notation to include the liminf function. $)
+  clsi $a class liminf $.
+
+  $d x k $.
+  $( Define the inferior limit of an infinite sequence of extended real
```

Comment:
In my last commit:
- an infinite sequence --> a sequence
- added (actually, it makes sense for any extended real function defined on a subset of RR which is not upper-bounded)
I like the additional comment, I wasn't so happy with "sequence" because the definition is more general than that.